ruby_event_store-browser/Makefile

```diff
@@ -34,4 +40,4 @@ clean-yarn:
 
 $(JS_BUNDLE):
 	@echo "Building browser frontend"
-	@cd elm; $(if $(CI), sysconfcpus -n 2 yarn build, yarn build)
\ No newline at end of file
+	@cd elm; $(if $(CI), sysconfcpus -n 2 yarn build, yarn build)
```

Comment:
In makefile, having a task named as `$(JS_BUNDLE)`, that is `public/ruby_event_store_browser.js`, means "proceed with steps below if there is no such file". This should work unchanged, although now we have 3 files instead of just 1 in output. I think `clean-yarn` task has to be aware of that.